boxmot/motion/adapters/botsort_kf_adapter.py

```diff
@@ -0,0 +1,161 @@
+import numpy as np
```

Comment:
`botsort`, `bytetrack` and `strongsort` have very similar KFs. So do `ocsort` and `deepocsort`. Any idea how this could be structured?

Reply:
They all use the same KF as `ocsort` and `deepocsort` now. The adapters are there to make that KF compatible with the original tracker implementations :-)